examples/webbilling-demo/src/tests/tax-calculation.test.ts

```diff
@@ -498,4 +431,52 @@ integrationTest.describe("Tax calculation", () => {
       await expect(calculateTaxesCount).toBe(2);
     },
   );
+
+  integrationTest("Stripe tax not active", async ({ page, userId }) => {
+    page = await navigateToTaxesLandingUrl(page, userId);
+
+    page.route("*/**/checkout/*/calculate_taxes", async (route) => {
+      await route.fulfill({
+        body: '{ "code": 7898, "message": "Stripe account setup error: Stripe Tax must be active to calculate taxes."}',
+        status: 422,
+      });
+    });
+
+    const packageCards = await getPackageCards(page);
+    await startPurchaseFlow(packageCards[0]);
+    await confirmPaymentError(page, "Stripe Tax not active");
+  });
+
+  integrationTest("Invalid tax origin address", async ({ page, userId }) => {
+    page = await navigateToTaxesLandingUrl(page, userId);
+
+    page.route("*/**/checkout/*/calculate_taxes", async (route) => {
+      await route.fulfill({
+        body: '{ "code": 7899, "message": "Stripe account setup error: Origin address for Stripe Tax is missing or invalid."}',
+        status: 422,
+      });
+    });
+
+    const packageCards = await getPackageCards(page);
+    await startPurchaseFlow(packageCards[0]);
+    const errorText = page.getByText("Invalid tax origin address", {
+      exact: true,
+    });
+    await expect(errorText).toBeVisible();
```

Comment:
Should we use `confirmPaymentError` like in the other two tests?

Reply:
Just tried, let's see if it passes tomorrow once the rate limit resets.